### DISCUSSION TITLE
Order of the option on the left menu

### DISCUSSION THREAD

**Discussion by joaobrazuna** [Leantime/leantime#945] [Ideas]
In my opinion, on the left menu the order of the options should be:
TO-Dos
Milestones
Timesheets
**Ideas
Research**
Retrospectives
Reports
Project Settings

(just because **Ideas** normally comes 1st, and only than comes **Research** )

Or better still:

TO-Dos
Milestones
Timesheets
**------------- (separator)**
**Ideas
Research**
Retrospectives
**------------- (separator)**
Reports
Project Settings

**Comment by marcelfolaron** [comment]
Thanks, this comes timely as we were just debating how to sinplify the
menu. Some thoughts that I would love to get your feedback on:

Dashboard

To-Dos <- Including a gantt view which can visualize todos + milestones

Strategy <- Research boards and Retros and other future types (risk log,
swot etc)

Ideas

Reports <- includes timesheets

Settings

What do you think?
